stablehlo/dialect/StablehloOps.td

```diff
@@ -1173,14 +1173,18 @@ def StableHLO_CaseOp: StableHLO_Op<"case", [
     ]> {
   let summary = "Switch-Case operator";
```

Comment:
Let's also delete the comment above `def StableHLO_CaseOp`. It is not relevant to StableHLO.